_code-samples/key-derivation/js/index.js

```diff
@@ -141,10 +172,10 @@ class Seed {
         // How often will an (essentially) random number generated by Sha512 be larger than that?
         // There's 2^32 chances (the for loop) to get a number smaller than the order,
         // and it's rare that you'll even get past the first loop iteration.
-        // Note that in TypeScript we actually need the throw, otherwise the function signature would be BN | undefined
+        // Note that in TypeScript we actually need the throw, otherwise the function signature would be bigint | undefined
```

Comment:
Wouldn't this be BigNumber?
```suggestion
        // Note that in TypeScript we actually need the throw, otherwise the function signature would be BigNumber | undefined
```